multisig/proposals/arbitrumone/vip-013/index.ts

```diff
@@ -0,0 +1,223 @@
+      params: [
+        "Liquid Staked ETH",
+        COMPTROLLER_LIQUID_STAKED_ETH,
+        parseUnits("0.5", 18),
+        parseUnits("1.02", 18),
+        parseUnits("100", 18),
+      ],
+    },
+    {
+      target: arbitrumone.VTREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
+      params: [wstETH, parseUnits("3.55", 18), arbitrumone.GUARDIAN],
+    },
+    {
+      target: wstETH,
+      signature: "approve(address,uint256)",
+      params: [arbitrumone.POOL_REGISTRY, 0],
+    },
+    {
+      target: wstETH,
+      signature: "approve(address,uint256)",
+      params: [arbitrumone.POOL_REGISTRY, parseUnits("3.55", 18)],
+    },
+    {
+      target: VwstETH,
+      signature: "setReduceReservesBlockDelta(uint256)",
+      params: ["86400"],
+    },
+    {
+      target: VwstETH,
+      signature: "setProtocolSeizeShare(uint256)",
+      params: [parseUnits("0.01", 18)],
+    },
+    {
+      target: arbitrumone.POOL_REGISTRY,
+      signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+      params: [
+        [
+          VwstETH,
+          parseUnits("0.93", 18),
+          parseUnits("0.95", 18),
+          parseUnits("3.55", 18),
+          vTokenReceiver,
+          parseUnits("8000", 18),
+          parseUnits("800", 18),
+        ],
+      ],
+    },
+    {
+      target: arbitrumone.RESILIENT_ORACLE,
+      signature: "setTokenConfig((address,address[3],bool[3]))",
+      params: [
+        [
+          weETH,
+          [
+            weETH_ONE_JUMP_ORACLE,
+            "0x0000000000000000000000000000000000000000",
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
+    {
+      target: arbitrumone.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[weETH, CHAINLINK_weETH_FEED, STALE_PERIOD_26H]],
+    },
+    {
+      target: arbitrumone.VTREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
+      params: [weETH, parseUnits("4", 18), arbitrumone.GUARDIAN],
+    },
+    {
+      target: weETH,
+      signature: "approve(address,uint256)",
+      params: [arbitrumone.POOL_REGISTRY, 0],
+    },
+    {
+      target: weETH,
+      signature: "approve(address,uint256)",
+      params: [arbitrumone.POOL_REGISTRY, parseUnits("4", 18)],
+    },
+    {
+      target: VweETH,
+      signature: "setReduceReservesBlockDelta(uint256)",
+      params: ["86400"],
+    },
+    {
+      target: VweETH,
+      signature: "setProtocolSeizeShare(uint256)",
+      params: [parseUnits("0.01", 18)],
+    },
+    {
+      target: arbitrumone.POOL_REGISTRY,
+      signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+      params: [
+        [
+          VweETH,
+          parseUnits("0.93", 18),
+          parseUnits("0.95", 18),
+          parseUnits("4", 18),
+          vTokenReceiver,
```

Comment:
This is wrong

Reply:
Refactored, b65668641d3c3d8bb1d533b80473ebccd781a8e3